tests/utils/helperfunctions.test.ts

```diff
@@ -0,0 +1,23 @@
+import { ImportTx } from "src/apis/evm"
+import { calcBytesCost } from "src/utils"
+
+describe("HelperFunctions", (): void => {
+  test("calcBytesCost", (): void => {
+    const importTx: ImportTx = new ImportTx()
+    // an empty EVM ImportTx is 76 bytes
+    let cost: number = 76
+    let bytesCost: number = calcBytesCost(importTx.toBuffer().byteLength)
+    expect(bytesCost).toEqual(cost)
+
+    // the byteCost should always be 1 nAVAX per byte
```

Comment:
Nit: the byteCost is 1 unit of gas per byte. This is not the same as nAVAX because it must be multiplied the baseFee to be put in terms of AVAX.

Reply:
Fixed in https://github.com/ava-labs/avalanchejs/commit/d919eab27b128638cf2307953d80361b7131676e